app/assets/javascripts/media_player_wrapper/mejs4_helper_quality.es6

```diff
@@ -33,18 +36,21 @@ class MEJSQualityHelper {
 
     for (let i = 0, total = radios.length; i < total; i++) {
       const radio = radios[i];
-      radio.addEventListener(
-        'change',
-        mejs4AvalonPlayer.mejsQualityHelper.sendQualitySelection
-      );
+      radio.addEventListener('change', this.updateQualitySelection.bind(this));
     }
   }
 
-  sendQualitySelection(e) {
+  updateQualitySelection(e) {
+    const quality = e.target.value;
+    mejs4AvalonPlayer.defaultQuality = quality;
+    this.sendQualitySelection(quality);
+  }
+
+  sendQualitySelection(quality) {
     $.ajax({
       type: 'POST',
       url: '/media_objects/set_session_quality',
-      data: { quality: e.target.value },
+      data: { quality: quality },
       dataType: 'json'
     });
```

Comment:
Does this need any type of callback where the UI needs to respond or do something?  Or is the POST request good enough to live on it's own?

Reply:
No, the post is good enough.